backend/main.py

```
@@ -18,14 +23,35 @@
+api = APIRouter(prefix="/api")
+
 # Simple hello endpoint
-@app.get("/api/hello")
+@app.get("/hello")
 async def hello():
     return {"message": "Hello, World!", "status": "success"}
 
 @app.get("/")
-async def root():
-    return {"message": "api works!"}
+def read_root(db: Session = Depends(get_db)):
```

Ruff (0.13.2)

Unused function argument: `db`

(ARG001)